Gruntfile.js

```diff
@@ -25,7 +27,8 @@ module.exports = function (grunt) {
                     {
                         expand: true,
                         flatten: true,
-                        src: ['node_modules/bootstrap/fonts/*'],
+                        src: ['node_modules/bootstrap/fonts/*',
```

Comment:
The array would be more readable if the elements were on different lines like the others. Makes for more readable code while following the code style of the repository.